v3 config update + new guide for agent tls term via config (#1118)
ngrok/ngrok-docs: docs/guides/other-guides/how-to-terminate-traffic-with-ngrok-configs.mdx

# Agent TLS Termination

> Agent tls termination allows you to secure your traffic with end-to-end encryption, seamlessly integrating it without needing to reconfigure your server. Even if the service you’re exposing doesn’t support TLS termination, you can still benefit from Zero-Knowledge TLS. The ngrok agent handles TLS termination for you. This approach is especially useful for legacy software that doesn’t support TLS or services operated outside your control.

Reviewer: ```suggestion
Agent TLS termination enables you to secure your traffic with end-to-end encryption without needing to reconfigure your server.

:::tip
If your service doesn’t support TLS termination, you can still use Agent TLS termination with [Zero-Knowledge TLS](/tls/tls-termination/#zero-knowledge-tls).
:::
```

I simplified this and linked to the relevant doc so we avoid repeating content. Repeated content hurts SEO and is hard to maintain (you might change one instance of the content and forget to update the other instances)
